src/fidesops/service/connectors/query_config.py

```diff
@@ -79,14 +71,16 @@ def filter_values(self, input_data: Dict[str, List[Any]]) -> Dict[str, Any]:
         Return a filtered list of key/value sets of data items that are both in
         the list of incoming edge fields, and contain data in the input data set
         """
-        return {
-            key: value
-            for (key, value) in input_data.items()
-            if key in self.query_keys
-            and isinstance(value, list)
-            and len(value)
-            and None not in value
-        }
+
+        out = {}
+        for key, values in input_data.items():
+            field = self.node.node.collection.field(key)
+            if field and key in self.query_keys and isinstance(values, list):
```

Comment:
why are we using `list` instead of `List` here, in the `isinstance` check?

Reply:
As far as I know the only difference is that `List` allows for generic typing. It seems that you get equivalent results for e.g.
 `isinstance([], list)` and `isinstance([], List)`